Add lint rule for max of practices slugs (#102)
exercism/docs: building/configlet/lint.md

- The `"exercises.practice[].practices"` values must be non-empty, lowercased strings using kebab-case
- The `"exercises.practice[].practices"` values must not have duplicates
- The `"exercises.practice[].practices"` values must match the `"concepts[].slug"` property of one of the concepts

> - The `"exercises.practice[].practices"` values must refer to a single concept's slug at most 10 times

Reviewer: ```suggestion
- The `"exercises.practice[].practices"` values must refer to an individual concept's slug at most 10 times (across all exercises)
```
